### ISSUE TITLE
Can't get karate.edg to work

### ISSUE THREAD

**Issue by RichardShea** [krishnanlab/PecanPy#207]
```bash
pecanpy --input demo/karate.edg --output demo/karate.emb --mode SparseOTF
```

Gives me the following error.

```txt
_Traceback (most recent call last):
  File "/usr/local/bin/pecanpy", line 5, in <module>
    from pecanpy.cli import main
  File "/usr/local/lib/python3.8/dist-packages/pecanpy/__init__.py", line 3, in <module>
    from . import pecanpy
  File "/usr/local/lib/python3.8/dist-packages/pecanpy/pecanpy.py", line 3, in <module>
    from gensim.models import Word2Vec
  File "/usr/local/lib/python3.8/dist-packages/gensim/__init__.py", line 11, in <module>
    from gensim import parsing, corpora, matutils, interfaces, models, similarities, utils  # noqa:F401
  File "/usr/local/lib/python3.8/dist-packages/gensim/corpora/__init__.py", line 6, in <module>
    from .indexedcorpus import IndexedCorpus  # noqa:F401 must appear before the other classes
  File "/usr/local/lib/python3.8/dist-packages/gensim/corpora/indexedcorpus.py", line 14, in <module>
    from gensim import interfaces, utils
  File "/usr/local/lib/python3.8/dist-packages/gensim/interfaces.py", line 19, in <module>
    from gensim import utils, matutils
  File "/usr/local/lib/python3.8/dist-packages/gensim/matutils.py", line 1030, in <module>
    from gensim._matutils import logsumexp, mean_absolute_difference, dirichlet_expectation
  File "gensim/_matutils.pyx", line 1, in init gensim._matutils
ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject_
```

**Comment by RemyLau** [comment]
Hi @RichardShea, thanks for using `pecanpy` and reporting the bug! This looks like an error caused by [`gensim`](https://github.com/RaRe-Technologies/gensim) due to [`numpy`](https://github.com/numpy/numpy). Could you share some information about the versions of these libraries you have? You could do so by running

```bash
pip show numpy gensim
```

I'm using `numpy==1.23.2` and `gensim==4.3.0` (Python 3.9 on Mac M1) and did not get the above error. I suspect the error would disappear if you upgrade your `numpy` and `gensim`. Here are some links that seem to be related to your issue (they all seem to indicate that a newer version of `numpy` should fix the error)

- https://stackoverflow.com/questions/66060487/valueerror-numpy-ndarray-size-changed-may-indicate-binary-incompatibility-exp
- https://github.com/numpy/numpy/issues/18709

**Comment by RichardShea** [comment]
Hi @RemyLau, I'm on gensim==4.3.0 and installed numpy==1.23.2.  I now get the following error:

_Traceback (most recent call last):
  File "/usr/local/bin/pecanpy", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/dist-packages/pecanpy/cli.py", line 342, in main
    g = read_graph(args)
  File "/usr/local/lib/python3.9/dist-packages/pecanpy/wrappers.py", line 18, in wrapper
    result = func(*args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/pecanpy/cli.py", line 294, in read_graph
    g.read_edg(path, weighted, directed, delimiter)
  File "/usr/local/lib/python3.9/dist-packages/pecanpy/graph.py", line 443, in read_edg
    g.read(path, weighted, directed, delimiter)
  File "/usr/local/lib/python3.9/dist-packages/pecanpy/graph.py", line 301, in read
    with open(path, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'demo/karate.edg'_

**Comment by RemyLau** [comment]
Hi @RichardShea, to run the example command for `demo/karate.edg`, you should be at the root level of the `pecanpy` repo directory. Let me know if this work!

**Comment by RichardShea** [comment]
@RemyLau 

Oh, I thought the file would be in the pip install....

I did a the `$ pip install pecanpy` in Google Colab.  I assumed the karate.edg file was in the library.  So now I cloned your repo after the pip install, cd to the pecanpy directory, and now it appears to run fine.   Here is my [Notebook](https://colab.research.google.com/drive/19CTuaOZ4NggJofZguefwSmRXjsZ4Q5tS?usp=sharing) if you are curious.  

**One Note:**
On https://github.com/krishnanlab/PecanPy, I think the `pecanpy --input demo/karate.edge --output demo/karate_n2vplus.emb --mode SparseOTF --extend` command should be `pecanpy --input demo/karate.edg --output demo/karate_n2vplus.emb --mode SparseOTF --extend`

Thank you for your help!
